### PR TITLE
Prepare release branch for v4.63.2 of `azure-cosmos` and v4.33.1 of `azure-cosmos-spark`.

### PR DESCRIPTION
**Pull request by jeet1995**
As title.

### CODE REVIEW

**Review by kushagraThapar — APPROVED** [review]
LGTM, thanks @jeet1995

File: sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md [hunk 1]
```diff
@@ -26,19 +26,19 @@ You can use any other Spark 3.1.1 spark offering as well, also you should be abl
 SLF4J is only needed if you plan to use logging, please also download an SLF4J binding which will link the SLF4J API with the logging implementation of your choice. See the [SLF4J user manual](https://www.slf4j.org/manual.html) for more information.
 
 For Spark 3.1:
-- Install Cosmos DB Spark Connector, in your spark Cluster [com.azure.cosmos.spark:azure-cosmos-spark_3-1_2-12:4.33.0](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-1_2-12/4.33.0/jar)
+- Install Cosmos DB Spark Connector, in your spark Cluster [com.azure.cosmos.spark:azure-cosmos-spark_3-1_2-12:4.33.1](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-1_2-12/4.33.0/jar)
```
Comment (FabianMeiswinkel) [inline sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md:29]:
```suggestion
- Install Cosmos DB Spark Connector, in your spark Cluster [com.azure.cosmos.spark:azure-cosmos-spark_3-1_2-12:4.33.1](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-1_2-12/4.33.1/jar)
```

File: sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md [hunk 2]
```diff
@@ -26,19 +26,19 @@ You can use any other Spark 3.1.1 spark offering as well, also you should be abl
...
 
 For Spark 3.2:
-- Install Cosmos DB Spark Connector, in your spark Cluster [com.azure.cosmos.spark:azure-cosmos-spark_3-2_2-12:4.33.0](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-2_2-12/4.33.0/jar)
+- Install Cosmos DB Spark Connector, in your spark Cluster [com.azure.cosmos.spark:azure-cosmos-spark_3-2_2-12:4.33.1](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-2_2-12/4.33.0/jar)
```
Comment (FabianMeiswinkel) [inline sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md:32]:
```suggestion
- Install Cosmos DB Spark Connector, in your spark Cluster [com.azure.cosmos.spark:azure-cosmos-spark_3-2_2-12:4.33.1](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-2_2-12/4.33.1/jar)
```

File: sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md [hunk 3]
```diff
@@ -26,19 +26,19 @@ You can use any other Spark 3.1.1 spark offering as well, also you should be abl
...
 
 For Spark 3.3:
-- Install Cosmos DB Spark Connector, in your spark Cluster [com.azure.cosmos.spark:azure-cosmos-spark_3-3_2-12:4.33.0](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-3_2-12/4.33.0/jar)
+- Install Cosmos DB Spark Connector, in your spark Cluster [com.azure.cosmos.spark:azure-cosmos-spark_3-3_2-12:4.33.1](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-3_2-12/4.33.0/jar)
```
Comment (FabianMeiswinkel) [inline sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md:35]:
```suggestion
- Install Cosmos DB Spark Connector, in your spark Cluster [com.azure.cosmos.spark:azure-cosmos-spark_3-3_2-12:4.33.1](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-3_2-12/4.33.1/jar)
```

File: sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md [hunk 4]
```diff
@@ -26,19 +26,19 @@ You can use any other Spark 3.1.1 spark offering as well, also you should be abl
...
   
 For Spark 3.4:
-- Install Cosmos DB Spark Connector, in your spark Cluster [com.azure.cosmos.spark:azure-cosmos-spark_3-4_2-12:4.33.0](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-4_2-12/4.33.0/jar)
+- Install Cosmos DB Spark Connector, in your spark Cluster [com.azure.cosmos.spark:azure-cosmos-spark_3-4_2-12:4.33.1](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-4_2-12/4.33.0/jar)
```
Comment (FabianMeiswinkel) [inline sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md:38]:
```suggestion
- Install Cosmos DB Spark Connector, in your spark Cluster [com.azure.cosmos.spark:azure-cosmos-spark_3-4_2-12:4.33.1](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-4_2-12/4.33.1/jar)
```

File: sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md [hunk 5]
```diff
@@ -26,19 +26,19 @@ You can use any other Spark 3.1.1 spark offering as well, also you should be abl
...
 
 For Spark 3.5:
-- Install Cosmos DB Spark Connector, in your spark Cluster [com.azure.cosmos.spark:azure-cosmos-spark_3-5_2-12:4.33.0](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-5_2-12/4.33.0/jar)
+- Install Cosmos DB Spark Connector, in your spark Cluster [com.azure.cosmos.spark:azure-cosmos-spark_3-5_2-12:4.33.1](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-5_2-12/4.33.0/jar)
```
Comment (FabianMeiswinkel) [inline sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md:41]:
```suggestion
- Install Cosmos DB Spark Connector, in your spark Cluster [com.azure.cosmos.spark:azure-cosmos-spark_3-5_2-12:4.33.1](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-5_2-12/4.33.1/jar)
```

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM

File: sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md (hunk 1 above)
Comment (jeet1995) [reply sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md]: Fixed in next iteration.

File: sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md (hunk 2 above)
Comment (jeet1995) [reply sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md]: Fixed in next iteration.

File: sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md (hunk 3 above)
Comment (jeet1995) [reply sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md]: Fixed in next iteration.

File: sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md (hunk 4 above)
Comment (jeet1995) [reply sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md]: Fixed in next iteration.

File: sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md (hunk 5 above)
Comment (jeet1995) [reply sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md]: Fixed in next iteration.

**Comment by jeet1995** [comment]
Corresponding CI run - https://dev.azure.com/azure-sdk/public/_build/results?buildId=4077247&view=results

**Pull request merged**